### PR TITLE
[Feature] Add tabs to swapbox

### PR DESCRIPTION
**Pull request by berteotti**
Screenshot 2022-02-14 at 10 26 23

### CODE REVIEW

File: src/pages/Swap/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+export const Tabs = () => {
+  return (
+    <TabsColumn>
+      <TabsRow>
+        <Button className="active">
+          <StyledEcoRouter />
+          Eco Router V1.5
+        </Button>
+        <Button disabled={true}>Limit</Button>
+        <Button disabled={true}>Bridge Swap</Button>
```
Comment (adamazad) [inline src/pages/Swap/Tabs.tsx:15]: Can we move these to translations?

File: src/pages/Swap/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  &.active {
+    color: #ffffff;
+    background: #2a2f42;
+    font-size: 12px;
+    line-height: 14px;
+  }
+
+  &:disabled {
+    color: #504d72;
```
Comment (adamazad) [inline src/pages/Swap/Tabs.tsx:58]: I believe these colors are defined the theme

File: src/pages/Swap/Tabs.tsx (hunk 1 above)
Comment (berteotti) [reply src/pages/Swap/Tabs.tsx:15]: Should Eco Router V1.5 be added too?

File: src/pages/Swap/Tabs.tsx (hunk 2 above)
Comment (berteotti) [reply src/pages/Swap/Tabs.tsx:58]: Found the above colors defined on the theme but not this last one. Should an issue be created? I can had it, but need to know the light mode equivalent.

Comment (adamazad) [reply src/pages/Swap/Tabs.tsx:58]: We can keep it here since it's just a single occurrence.

Comment (berteotti) [reply src/pages/Swap/Tabs.tsx:58]: It's being used twice, in Bridge/Tabs also.

Comment (adamazad) [reply src/pages/Swap/Tabs.tsx:58]: Ok, let's move it to the `theme` then.

Comment (berteotti) [reply src/pages/Swap/Tabs.tsx:58]: Will fix Bridge/Tabs theme colors too in these PR. Where can I find light mode version to `#504d72`?

**Pull request merged**